lib/k8s-test-framework/src/version.rs

```diff
@@ -0,0 +1,70 @@
+//! Perform a version lookup.
+use super::Result;
+
+use std::process::Stdio;
+use tokio::process::Command;
+
+/// Exec a `kubectl` command to pull down the kubernetes version
+/// metadata for a running cluster for use in the test framework
+pub async fn get_version(kubectl_command: &str) -> Result<K8sVersion> {
```

Comment:
```suggestion
pub async fn kubernetes_version(kubectl_command: &str) -> Result<K8sVersion> {
```

Reply:
So I think if we change the module name to `kubernetes_version` we might want to keep the function as `get_version` so we end up with a function signature of `kubernetes_version::get_version()` which i think is more descriptive than `kubernetes_version::kubernetes_version`. Do you feel ok with that?

Reply:
What I was thinking is this looks a lot like

https://github.com/timberio/vector/blob/b401de37aacb16d189fe0aa93ad5d202c80ddcbc/lib/k8s-test-framework/src/lib.rs#L24

which we then reexport as

https://github.com/timberio/vector/blob/b401de37aacb16d189fe0aa93ad5d202c80ddcbc/lib/k8s-test-framework/src/lib.rs#L42

for better usability.

In this case, we probably can't get away wich such reexport, as we're also exporting the `K8sVersion` - sp yeah, I guess it is fine.

We can reduce it even further to `get`, so the usage will be `kubernetes_version::get()`, and the type would be addressed `kubernetes_version::K8sVersion`.

Reply:
Done!